proposals/credential-management.md

```diff
@@ -0,0 +1,128 @@
+# browser.credentials
+
+WebExtension proposal to allow interfacing with the [Credential Management API](https://w3c.github.io/webappsec-credential-management)
+
+## Motivation
+
+Password manager extensions currently rely on a mixture of heuristics and API overriding to detect and store credentials.
+To fill credentials they also rely on DOM APIs to detect relevant input fields to display UI (or autofill).
+
+Browsers have APIs for interaction with various credential types, most notably PublicKey and Password (Chromium only). However, these aren't exposed to browser extensions.
+
+## Proposal
+
+We propose a new browser.credentials API that would allow extensions to integrate with these existing web APIs.
```

Comment:
Need more detail on what the expected integration looks like from a UI /UX perspective. Potentially a UI mock could be useful but the exact details rely on unknowns such as whether to have 1 credential manager or multiple based on credential type.